app/controllers/user_sessions_controller.rb

```diff
@@ -1,6 +1,7 @@
 class UserSessionsController < ApplicationController
   before_action :require_no_user, only: [:new]
   def new
+    @user_session = UserSession.new
```

Comment:
It seems like this wasn't specified in the original issue? Did you take a look at the user sessions controller [here](https://github.com/publiclab/plots2/blob/main/app/controllers/user_sessions_controller.rb)?

Sorry I can't be more helpful, this isn't really my expertise, but perhaps @cesswairimu @jywarren or @sagarpreet-chadha could help.

Reply:
Yeah I added that in after noticing the tests were failing. Thanks a lot for looking into this btw. I'll post the stack trace as well